src/legacy_wrappers/mongo_client.js

```diff
@@ -7,6 +7,18 @@ Object.defineProperty(module.exports, '__esModule', { value: true });
 
 module.exports.makeLegacyMongoClient = function (baseClass) {
   class LegacyMongoClient extends baseClass {
+    // constructor adds client metadata before constructing final client
+    constructor(connectionString, options) {
+      options = { ...options };
+      if (options.driverInfo != null && typeof options.driverInfo.name === 'string') {
```

Comment:
Since this is being released separately (I think), should we also include its version as well as its name?

Reply:
Yep it is to be released separately, added the version